egui/src/widgets/plotting/plot.rs

```diff
@@ -244,7 +22,7 @@ struct PlotMemory {
 ///
 /// ```
 /// # let ui = &mut egui::Ui::__test();
-/// use egui::plot::{Curve, Plot, Value};
+/// use egui::plotting::{Curve, Plot, Value};
```

Comment:
This rename will be a breaking change for users. I think we should just call the module `plot` instead to avoid this.